examples/specific/c_file.h

```diff
@@ -26,7 +26,7 @@ struct cache_tree {
 extern struct cache_tree global_cache_tree;
 
 struct cache_tree *cache_tree(void);
-void cache_tree_free(struct cache_tree **);
+extern void cache_tree_free(struct cache_tree **);
```

Comment:
How come it's ``extern``?

Reply:
Because it is used for the extern example in the docs. I changed it to use this function, because there were undefined type used as parameters in the original choice. These undefined types caused warnings in rendering the function signature.

Reply:
Ah, makes sense.